modules/stitching/include/opencv2/stitching.hpp

```diff
@@ -286,7 +286,8 @@ class CV_EXPORTS_W Stitcher
     CV_WRAP Status stitch(InputArrayOfArrays images, InputArrayOfArrays masks, OutputArray pano);
 
     std::vector<int> component() const { return indices_; }
-    std::vector<detail::CameraParams> cameras() const { return cameras_; }
+    CV_WRAP std::vector<CameraParams> getCameras() const { return cameras_; }
```

Comment:
please restore original method signature and remove `CV_WRAP` prefix. It cannot be handled by bindings generator automatically.